cmd/wallet-web/src/pages/chapi/Signup.vue

```diff
@@ -2,15 +2,15 @@
   <div class="bg-neutrals-softWhite flex  min-w-screen min-h-screen items-center justify-center px-5 py-5
               bg-scroll lg:bg-onboarding xl:bg-onboarding xs:bg-onboarding sm:bg-onboarding 2xl:bg-onboarding md:bg-onboarding bg-no-repeat">
     <div class="bg-gradient-dark rounded-xl overflow-hidden text-neutrals-softWhite text-xl md:text-3xl
-    2xl:h-xl 2xl:w-xl xl:h-xl xl:w-xl sm:h-xl sm:w-xl md:h-xl md:w-xl h-fit-content w-full -mt-40 sm:-mt-56 xs:-mt-56 lg:mt-0 2xl:mt-0 xl:mt-0 md:mt-10">
+    2xl:h-full 2xl:w-xl xl:h-full xl:w-xl sm:h-full sm:w-xl md:h-full md:w-xl w-full -mt-40 sm:-mt-56 xs:-mt-56 lg:mt-0 2xl:mt-0 xl:mt-0 md:mt-10">
       <!--Trustbloc Intro div  -->
       <div class="grid 2xl:grid-cols-2 lg:grid-cols-2 md:grid-cols-2 sm:grid-cols-1 xs:grid-cols-1
-           bg-flare bg-no-repeat w-auto h-auto lg:divide-x md:divide-x divide-neutrals-medium divide-opacity-25">
+           bg-flare bg-no-repeat w-auto h-3/5 divide-x divide-neutrals-medium divide-opacity-25">
         <div class="col-span-1 hidden xl:block 2xl:block lg:block md:block py-14 px-8">
           <ul class="list-none md:list-disc text-sm px-4 py-4">
             <div class="flex items-center">
               <img class="h-2 w-8 mr-2 mt-4 md-4 my-2 mx-4" src="@/assets/img/logo.svg">
-              <h2 class="font-bold text-4xl lg:text-3xl">TrustBloc</h2>
+              <h2 class="font-bold text-4xl lg:text-6xl">TrustBloc</h2>
```

Comment:
The styles for `h2` and other headings should already come from the theme, including the font weight, font size and line height. You should be able to remove these classes from here.

Reply:
This was tracked in #807 but i will see if i can push in this pr

Reply:
We fill integrate theme in the follow up pr, meanwhile @birtony can verify some discrepancies with the theme

Reply:
@birtony  will verify couple of discrepancies with the theme and then will update in the #807 pr